### DISCUSSION TITLE
Cannot read property 'jquery' of null

### DISCUSSION THREAD

**Discussion by lucasgama335** [orchidjs/tom-select#194] [Q&A]
i'm using tom-select together with laravel when developing a project, everything works fine, but i always get the following error on the console. And as much as this error apparently does not affect the operation of the code, having it there bothers me.

```
Uncaught TypeError: Cannot read property 'jquery' of null
    at getDom (vanilla.ts:10)
    at new TomSelect (tom-select.ts:96)
    at plugin.ts:20
```

My WebPack Config:

```
// Tom Select Imports
.combine([
    'node_modules/tom-select/dist/js/tom-select.complete.js',
    'resources/js/vendor/tom-select.js' // Create Tom-Select Elements
], 'public/js/vendor/tom-select/tom-select.js')
```

Elements:

```
// Patients RelationShip With Users
const patientElement = document.querySelector('#usersSelect');
if (patientElement) {
    new TomSelect(patientElement, {
        plugins:['remove_button'],
        persist: false,
        create: true,
        maxOptions: 100,
        sortField: {
            field: "text",
            direction: "asc"
        }
    });
}
```

Does anyone know what might be causing this error? I don't use jquery anywhere in my project.

**Comment by oyejorge** [comment]
Which version of Tom Select are you using?

**Reply by lucasgama335** [reply]
1.7.8

**Reply by oyejorge** [reply]
I haven't been able to reproduce the error. [Do you see it with this example](https://jsfiddle.net/8yhj1nfe/)

**Comment by cadu-leite** [comment]
same proble here 

 - tom-select@2.3.1

I'm not using JQuery at all 

and using - 
`<script src="https://cdn.jsdelivr.net/npm/tom-select@2.3.1/dist/js/tom-select.complete.min.js"></script>`

```
    vanilla.ts:12 Uncaught TypeError: Cannot read properties of null (reading 'jquery')
    at j (vanilla.ts:12:12)
    at new de (tom-select.ts:101:18)
    at setSelect (7756efab-9a32-46e5-9ce0-1d0677065632/:872:19)
    at <anonymous>:9:4
    at HTMLBodyElement.<anonymous> (htmx_1.9.10.min.js:1:5448)
    at ce (htmx_1.9.10.min.js:1:26755)
    at htmx_1.9.10.min.js:1:10321
    at htmx_1.9.10.min.js:1:44741
    at oe (htmx_1.9.10.min.js:1:4604)
    at s (htmx_1.9.10.min.js:1:44716)
```
still trying to get more details/understandi

**Comment by sistemas17peru** [comment]
Focus on the element. In my case, the select-element was null. It was my mistake for not noticing it earlier. I spent quite some time debugging, but that was basically the issue.

`const selectForm = document.getElementById('select-element');`
`if (!selectForm) return;`
